wfe/wfe.go

```diff
@@ -687,7 +687,12 @@ func (wfe *WebFrontEndImpl) NewAuthorization(ctx context.Context, logEvent *web.
 	logEvent.Created = authz.ID
 
 	// Make a URL for this authz, then blow away the ID and RegID before serializing
-	authzURL := web.RelativeEndpoint(request, authzPath+string(authz.ID))
+	var authzURL string
+	if authz.V2 {
+		authzURL = web.RelativeEndpoint(request, fmt.Sprintf("%sv2/%s", authzPath, authz.ID))
```

Comment:
maybe worth making an `authzPath2` constant to use here to concat with `string(authz.ID)` instead of the sprintf with the `authzPath`? The lone `2` in the format string seems easy to miss.